lib/faker/date.ex

```diff
@@ -35,6 +35,22 @@ if Version.match?(System.version(), ">= 1.3.0") do
     end
 
     @doc """
+    Returns a random date in the past up to today in NaiveDateTime
```

Comment:
Consider changing this line of doc to something like `Returns a random date between the given date in the past and today, today (not?) included`